randnum.c

```diff
@@ -0,0 +1,18 @@
+#include<stdio.h>
+#include<stdlib.h>
+#include<time.h>
+
+int main(){
+	srand(time(NULL));
+	int r = rand()%11;
```

Comment:
You were incredibly close! Your range included 0, which it wasn't supposed to do. To fix this, you should have written int r = rand()%10 +1; . Besides that, nice job being on the right track towards an efficient program!